src/inject/dynamic-theme/modify-css.ts

```diff
@@ -35,17 +35,15 @@ export function getModifiableCSSDeclaration(
     ignoreImageSelectors: string[],
     isCancelled: () => boolean,
 ): ModifiableCSSDeclaration {
-    const important = Boolean(rule && rule.style && rule.style.getPropertyPriority(property));
-    const sourceValue = value;
     if (property.startsWith('--')) {
         const modifier = getVariableModifier(variablesStore, property, value, rule, ignoreImageSelectors, isCancelled);
         if (modifier) {
-            return {property, value: modifier, important, sourceValue};
+            return {property, value: modifier, important: Boolean(rule && rule.style && rule.style.getPropertyPriority(property)), sourceValue: value};
```

Comment:
Great, now I think you should move it to some function, to not repeat the same line.